cirq/sim/clifford/clifford_simulator.py

```diff
@@ -354,6 +372,7 @@ def apply_unitary(self, op: 'cirq.Operation'):
         rewrite=lambda args, kwargs: (args, {('collapse_state_vector' if k ==
                                               'collapse_wavefunction' else k): v
                                              for k, v in kwargs.items()}))
+    @deprecated(deadline='v0.10.0', fix='Use the apply_measurement instead')
```

Comment:
Is the `CliffordState` directly used by end users? Do they call the `perform_measurement` function? If not, I can just delete it.

Reply:
Let's just be on the safe side, it's a public function. We don't have usage numbers!